core/src/main/java/org/apache/calcite/plan/hep/HepPlanner.java

```diff
@@ -186,14 +186,14 @@ public HepPlanner(
   }
 
   @Override public RelNode findBestExp() {
-    requireNonNull(root, "root");
+    requireNonNull(root, "'root' must not be null");
```

Comment:
What is the error message before this PR? I want to know `must not be null` it is repetitive with `requireNonNull`? The other places in this project all use it like `requireNonNull(root, 'root');`

Reply:
Ok, thanks for reviewing the code for me. I was initially trying to get the same error message from findBestExp for HepPlanner and VolcanoPlanner
![image](https://github.com/user-attachments/assets/62a28cef-99e1-4bda-a63a-416bbf63813f)

Reply:
Ok, thanks for reviewing the code for me